elixir/apps/domain/lib/domain/config.ex

```diff
@@ -113,10 +113,20 @@ defmodule Domain.Config do
     end
   end
 
+  ## Feature flag helpers
+
   def sign_up_enabled? do
     compile_config!(Definitions, :feature_sign_up_enabled)
   end
 
+  def flows_enabled? do
```

Comment:
Flows are enabled but this feature is hiding the activities (for which connlib doesn't send the data yet)
```suggestion
  def flow_activities_enabled? do
```